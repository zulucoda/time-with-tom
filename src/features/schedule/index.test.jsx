import { render } from '@testing-library/react';
import Schedule from '.';
import React from 'react';

describe('Features - Schedule - Unit Test', function () {
  test('does it render', () => {
    const { container } = render(<Schedule />);
    expect(container).toMatchSnapshot();
  });
});
