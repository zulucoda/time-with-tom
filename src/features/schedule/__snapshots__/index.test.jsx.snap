// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features - Schedule - Unit Test does it render 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthMd"
    >
      <div
        class="MuiBox-root MuiBox-root-12"
      >
        <div
          class="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom MuiTypography-alignCenter"
            marked="center"
          >
            Schedule
          </h3>
          <table
            class="MuiTable-root WithStyles(ForwardRef(Table))-root-96"
          >
            <thead
              class="MuiTableHead-root WithStyles(ForwardRef(TableHead))-root-99"
            >
              <tr
                class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-101 MuiTableRow-head"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-108"
                  scope="col"
                >
                  Name
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-121"
                  scope="col"
                >
                  Surname
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-122"
                  scope="col"
                >
                  Email
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-123"
                  scope="col"
                >
                  Activity
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-124"
                  scope="col"
                >
                  Date
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-125"
                  scope="col"
                >
                  Start Time
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-126"
                  scope="col"
                >
                  End Time
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root WithStyles(ForwardRef(TableBody))-root-127"
            >
              <tr
                class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-129"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-131"
                >
                  Muzi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-132"
                >
                  Buthelezi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-133"
                >
                  muzi@mfbproject.co.za
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-134"
                >
                  squash
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-135"
                >
                  2020-04-22
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-136"
                >
                  09:00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-137"
                >
                  13:00
                </td>
              </tr>
              <tr
                class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-129"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-138"
                >
                  Muzi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-139"
                >
                  Buthelezi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-140"
                >
                  muzi@mfbproject.co.za
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-141"
                >
                  squash
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-142"
                >
                  2020-04-23
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-143"
                >
                  09:00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-144"
                >
                  13:00
                </td>
              </tr>
              <tr
                class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-129"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-145"
                >
                  Muzi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-146"
                >
                  Buthelezi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-147"
                >
                  muzi@mfbproject.co.za
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-148"
                >
                  squash
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-149"
                >
                  2020-04-24
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-150"
                >
                  09:00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-151"
                >
                  13:00
                </td>
              </tr>
              <tr
                class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-129"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-152"
                >
                  Muzi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-153"
                >
                  Buthelezi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-154"
                >
                  muzi@mfbproject.co.za
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-155"
                >
                  squash
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-156"
                >
                  2020-04-25
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-157"
                >
                  09:00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-158"
                >
                  13:00
                </td>
              </tr>
              <tr
                class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-129"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-159"
                >
                  Muzi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-160"
                >
                  Buthelezi
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-161"
                >
                  muzi@mfbproject.co.za
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-162"
                >
                  squash
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-163"
                >
                  2020-04-26
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-164"
                >
                  09:00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-130 WithStyles(ForwardRef(TableCell))-root-165"
                >
                  13:00
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
